JOSS/paper.md

```diff
@@ -0,0 +1,132 @@
+---
+title: 'SSAPy - Space Situational Awareness for Python'
+tags:
+  - Python
+  - space domain awareness
+  - orbits
+  - cislunar space
+authors:
+  - name: Joshua E. Meyers
+    affiliation: [1, 2]
+    orcid: 0000-0002-2308-4230
+  - name: Michael D. Schneider
+    affiliation: 3
+    orcid: 0000-0002-8505-7094
+  - name: Julia T. Ebert
+    affiliation: 4
+    orcid: 0000-0002-1975-772X
+  - name: Edward F. Schlafly
+    affiliation: 5
+    orcid: 0000-0002-3569-7421
+  - name: Travis Yeager
+    affiliation: 3
+    orcid: 0000-0002-2582-0190
+  - name: Alexx Perloff
+    affiliation: 3
+    orcid: 0000-0001-5230-0396
+  - name: Daniel Merl
+    affiliation: 3
+    orcid: 0000-0003-4196-5354
+  - name: Noah Lifset
+    affiliation: 6
+    orcid: 0000-0003-3397-7021
+  - name: Jason Bernstein
+    affiliation: 3
+    orcid: 0000-0002-3391-5931
+  - name: William A. Dawson
+    affiliation: 3
+    orcid: 0000-0003-0248-6123
+  - name: Nathan Golovich
+    affiliation: 3
+    orcid: 0000-0003-2632-572X
+  - name: Denvir Higgins
+    affiliation: 3
+    orcid: 0000-0002-7579-1092
+  - name: Peter McGill
+    affiliation: 3
+    orcid: 0000-0002-1052-6749
+    corresponding: true
+  - name: Caleb Miller
+    affiliation: 3
+    orcid: 0000-0001-6249-0031
+  - name: Kerianne Pruett
+    affiliation: 3
+    orcid: 0000-0002-2911-8657
+affiliations:
+ - name: SLAC National Accelerator Laboratory, 2575 Sand Hill Road, Menlo Park, CA 94025, USA
+   index: 1
+ - name: Kavli Institute for Particle Astrophysics and Cosmology, Stanford University, 452 Lomita Mall, Stanford, CA 94035, USA
+   index: 2
+ - name: Lawrence Livermore National Laboratory, 7000 East Ave., Livermore, CA 94550, USA
+   index: 3
+ - name: Fleet Robotics, 21 Properzi Way, Somerville, MA 02143, USA
+   index: 4
+ - name: Space Telescope Science Institute, 3700 San Martin Drive, Baltimore, MD 21218, USA
+   index: 5
+ - name: University of Texas at Austin, 2515 Speedway, Austin, TX 78712, USA
+   index: 6
+  
+
+
+date: 13 August 2017
```

Comment:
Why is this date in 2017?

Reply:
default in the JOSS mark down, does not matter for JOSS submission. I'll consider this resolved unless you suggest a non default date.

Reply:
Just put today's date.